### ISSUE TITLE
A question about focal loss

### ISSUE THREAD

**Issue by PengxinWang** [ilnehc96/HKU-DASC7606-A1#3]
In losses.py, we're required to finish the part of Focal Loss from 107th line to 117th line. 

`focal_weight = "?"`

`bce = "?"`

`cls_loss = focal_weight * bce`

However, I'm a little bit confused about bce since we are handling multiclasses. Are we supposed to use cce(categorical cross entropy) instead? 

Another question is about evaluation, since the evaluation metric (the AP score) is wrapped in COCOeval, I can't see very clearly. I want to ask, does the model's performance on categorical classification is taken into consideration in evaluation? Also, does model's ability of correctly classify the category will influence the performance of correctly positioning the object? 

Really thanks!!

**Comment by PengxinWang** [comment]
It confused me a lot about the intepretation of classification loss because I formerlly thought the output layer head of classification head is a softmax layer... Until I find similar question on the [stackoverflow](https://stackoverflow.com/questions/53809995/confusions-regarding-retinanet). In case others get similar confusion. I post this issue.
